services/src/main/java/org/keycloak/authentication/authenticators/resetcred/ResetPassword.java

```diff
@@ -18,9 +18,12 @@
 package org.keycloak.authentication.authenticators.resetcred;
 
 import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.authentication.requiredactions.UpdatePassword;
```

Comment:
Looks like unrelated but I'm not hard on that :)

We really need to enable code style/format checks ...